src/main/java/utils/SeleniumDriver.java

```diff
@@ -32,7 +32,7 @@
         WebDriverManager.chromedriver().setup();
         ChromeOptions chromeOptions = new ChromeOptions();
         chromeOptions.addArguments("--no-sandbox");
-        chromeOptions.addArguments("--headless");
+         chromeOptions.addArguments("--headless");
```

Comment:
looks like incidental change

Reply:
Yes every-time to check on local i need to comment headless. Space got committed.